openadapt/config.py

```diff
@@ -1,135 +1,34 @@
-    "RECORD_WINDOW_DATA": False,
-    # may incur significant performance penalty
-    "RECORD_READ_ACTIVE_ELEMENT_STATE": False,
-    # TODO: remove?
-    "REPLAY_STRIP_ELEMENT_STATE": True,
-    # IGNORES WARNINGS (PICKLING, ETC.)
-    # TODO: ignore warnings by default on GUI
-    "IGNORE_WARNINGS": False,
-    "MAX_NUM_WARNINGS_PER_SECOND": 5,
-    "WARNING_SUPPRESSION_PERIOD": 1,
-    "MESSAGES_TO_FILTER": ["Cannot pickle Objective-C objects"],
-    # ACTION EVENT CONFIGURATIONS
-    "ACTION_TEXT_SEP": "-",
-    "ACTION_TEXT_NAME_PREFIX": "<",
-    "ACTION_TEXT_NAME_SUFFIX": ">",
-    # PERFORMANCE PLOTTING CONFIGURATION
-    "PLOT_PERFORMANCE": True,
-    # CAPTURE CONFIGURATION
-    "CAPTURE_DIR_PATH": "captures",
-    # APP CONFIGURATIONS
-    "APP_DARK_MODE": False,
-    # SCRUBBING CONFIGURATIONS
-    "SCRUB_ENABLED": False,
-    "SCRUB_CHAR": "*",
-    "SCRUB_LANGUAGE": "en",
-    # TODO support lists in getenv_fallback
-    "SCRUB_FILL_COLOR": 0x0000FF,  # BGR format
-    "SCRUB_CONFIG_TRF": {
-        "nlp_engine_name": "spacy",
-        "models": [{"lang_code": "en", "model_name": "en_core_web_trf"}],
-    },
-    "SCRUB_PRESIDIO_IGNORE_ENTITIES": [
-        # 'US_PASSPORT',
-        # 'US_DRIVER_LICENSE',
-        # 'CRYPTO',
-        # 'UK_NHS',
-        # 'PERSON',
-        # 'CREDIT_CARD',
-        # 'US_BANK_NUMBER',
-        # 'PHONE_NUMBER',
-        # 'US_ITIN',
-        # 'AU_ABN',
-        # 'DATE_TIME',
-        # 'NRP',
-        # 'SG_NRIC_FIN',
-        # 'AU_ACN',
-        # 'IP_ADDRESS',
-        # 'EMAIL_ADDRESS',
-        # 'URL',
-        # 'IBAN_CODE',
-        # 'AU_TFN',
-        # 'LOCATION',
-        # 'AU_MEDICARE',
-        # 'US_SSN',
-        # 'MEDICAL_LICENSE'
-    ],
-    "SCRUB_KEYS_HTML": [
-        "text",
-        "canonical_text",
-        "title",
-        "state",
-        "task_description",
-        "key_char",
-        "canonical_key_char",
-        "key_vk",
-        "children",
-    ],
-    # VISUALIZATION CONFIGURATIONS
-    "VISUALIZE_DARK_MODE": False,
-    "VISUALIZE_RUN_NATIVELY": True,
-    "VISUALIZE_DENSE_TREES": True,
-    "VISUALIZE_ANIMATIONS": True,
-    "VISUALIZE_EXPAND_ALL": False,  # not recommended for large trees
-    "VISUALIZE_MAX_TABLE_CHILDREN": 10,
-    # Calculate and save the difference between 2 neighboring screenshots
-    "SAVE_SCREENSHOT_DIFF": False,
-    "SPACY_MODEL_NAME": "en_core_web_trf",
-    "PRIVATE_AI_API_KEY": "<set your api key in .env>",
-    "RECORD_VIDEO": False,
-    "RECORD_IMAGES": True,
-    "VIDEO_PIXEL_FORMAT": "rgb24",
-    "DASHBOARD_CLIENT_PORT": 3000,
-    "DASHBOARD_SERVER_PORT": 8000,
-}
-
-# each string in STOP_STRS should only contain strings
-# that don't contain special characters
+from openadapt.build_utils import (
+    get_path_to_preferences_folder,
+    is_running_from_executable,
+)
+
+CURRENT_DIRPATH = pathlib.Path(__file__).parent
+PREFERENCES_DIRPATH = get_path_to_preferences_folder(CURRENT_DIRPATH)
```

Comment:
While we're here, it appears we mix the use of `DIRPATH` and `DIR_PATH`. Let's stick with `DIR_PATH` and make it consistent everywhere.